stable/traefik/values.yaml

```diff
@@ -1,6 +1,8 @@
 ## Default values for Traefik
 image: traefik
 imageTag: 1.4.6
+## can switch the deploy type to DaemonSet if you prefer
+deployType: Deployment
```

Comment:
If you can document this option in the table of config options found in the README, this will LGTM.

Reply:
Ok, pushed